staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -220,13 +262,24 @@ func TestDifferentFlowsWithoutQueuing(t *testing.T) {
 	}
 	qs := qsc.Complete(fq.DispatchingConfig{ConcurrencyLimit: 4})
 
-	exerciseQueueSetUniformScenario(t, "DifferentFlowsWithoutQueuing", qs, []uniformClient{
+	exerciseQueueSetUniformScenario(t, qCfg.Name, qs, []uniformClient{
 		{1001001001, 6, 10, time.Second, 57 * time.Millisecond},
 		{2002002002, 4, 15, time.Second, 750 * time.Millisecond},
-	}, time.Second*13, false, false, clk, counter)
+	}, time.Second*13, false, false, false, true, clk, counter)
+	err = metrics.GatherAndCompare(`
```

Comment:
Is this really how people normally test metrics? I really would have expected to read the desired values and compare ints/floats. I'd also be satisfied with just reading the gauge directly if that's possible. @logicalhan

Reply:
I found this pattern when looking at other metrics_test.go files in k8s.io/apiserver.